tests/test-cases/check-atmos-help-in-empty-dir.yaml

```
@@ -0,0 +1,15 @@
+description: "Test 'atmos --help' command in empty directory"
+workdir: "fixtures/scenarios/empty-dir"
+command: "atmos"
+args:
+  - --help
+expect:
+  diff:
+    - "──────────────────────────────────────────────────────────────"
+    - "Update available!"
+    - "Atmos Releases:"
+    - "Install Atmos:"
+  exit_code: 0 
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)